src/makeInfinityAdder.js

```diff
@@ -4,7 +4,25 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  const adder = (number) => {
+    if (number === undefined) {
```

Comment:
```suggestion
    if (!number) {
```

Reply:
hello, no. You unright in this moment. It should to left this condition. When function get 0 it understand him as a boolean, not like a number. What's why appeared a mistake. Now I need no add adding condition to do it works. I've left this condition as I do it at first.